File: app/src/main/java/com/durbar/bangabandhuplay/ui/family_member/FamilyMemberViewModel.java
package com.durbar.bangabandhuplay.ui.family_member;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;
import androidx.lifecycle.MutableLiveData;

import com.durbar.bangabandhuplay.data.model.family_member.Data;
import com.durbar.bangabandhuplay.data.repository.FamilyMemberRepository;

import java.util.List;

public class FamilyMemberViewModel extends AndroidViewModel {

    private FamilyMemberRepository repository;
    public FamilyMemberViewModel(@NonNull Application application) {
        super(application);
        repository = new FamilyMemberRepository(application);
    }

    public MutableLiveData<List<Data>> fetchFamilyMemberPhotos(){
        return repository.fetchFamilyMemberPhotos();
    }
}

File: app/src/main/java/com/durbar/bangabandhuplay/ui/family_member/FamilyMemberViewModel.kt
package com.durbar.bangabandhuplay.ui.family_member

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import androidx.lifecycle.MutableLiveData
import com.durbar.bangabandhuplay.data.model.family_member.Data
import com.durbar.bangabandhuplay.data.repository.FamilyMemberRepository

class FamilyMemberViewModel(application: Application) : AndroidViewModel(application) {
    private val repository: FamilyMemberRepository

    init {
        repository = FamilyMemberRepository(application)
    }

    fun fetchFamilyMemberPhotos(): MutableLiveData<List<Data>> {
        return repository.fetchFamilyMemberPhotos()
    }
}
